dist/formio.js

```diff
@@ -39,10 +39,10 @@ require('whatwg-fetch');
 var EventEmitter = require('eventemitter2').EventEmitter2;
 var copy = require('shallow-copy');
 
-/**
- * The Formio interface class.
- *
- *   let formio = new Formio('https://examples.form.io/example');
```

Comment:
Why were all of these changed? Do you have a setting in your IDE that is changing each of these comments? Please fix before I can accept this pull request. Thank you for your contributions!